deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -18,7 +18,7 @@ rules:
     verbs: ["list", "watch", "create", "update", "patch"]
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "create", "delete", "patch"]
+    verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
```

Comment:
Are you sure this is needed ? 
rbac from external-provisioner does not need it.
https://github.com/kubernetes-csi/external-provisioner/blob/b67655857d4800483928fe62e17660175bcb6ef0/deploy/kubernetes/rbac.yaml#L30

Reply:
yes, the ci was failing earlier with this error. ```failed to remove finalizer for persistentvolume: persistentvolumes "pvc-1acd6d4a-67d0-48da-b942-757f7970258a" is forbidden: User "system:serviceaccount:cephcsi-e2e-8301cf6b:cephfs-csi-provisioner" cannot update resource "persistentvolumes" in API group "" at the cluster scope```